packages/circuit-ui/components/Table/Table.tsx

```diff
@@ -103,18 +102,20 @@ const tableContainerScrollableStyles = ({
     }
   `;
 
-const noShadowStyles = ({ noShadow }: TableContainerElProps) =>
-  noShadow &&
+const shadowStyles = ({
+  theme,
+  noShadow,
```

Comment:
Should we deprecate this for v3 and rename it `noBorder`? What do you think?

(or a better name, can't think of one right now)

Reply:
I didn't quite understand when the `noShadow` prop would be used, so I didn't want to change it. Perhaps we don't need it anymore at all.

Reply:
It's when the table doesn't have an outer outline. The most common use case seems to be when it's used on a Card: a Table outline would "duplicate" the Card's outline. Here's an example from Subscriptions, I found 5 occurrences in the Dashboard:

<img width="882" alt="Screenshot 2021-06-04 at 11 16 09" src="https://user-images.githubusercontent.com/35560568/120778204-5b87fd00-c526-11eb-9954-11b2fed05572.png">

Reply:
Maybe there's a better name for it would be `noOutline` then, or even better, maybe we should remove the outline altogether and only use the Table on a Card.

I think this can be a separate PR, though, and it can probably be grouped with the upcoming Table redesign–let's keep it like this for now.

Reply:
> it can probably be grouped with the upcoming Table redesign

I agree.